src/handlers.rs

```diff
@@ -10,21 +10,13 @@ use std::fmt;
 use std::sync::Arc;
 use tracing as log;
 
-#[derive(Debug)]
-pub enum HandlerError {
-    Message(String),
-    Other(anyhow::Error),
-}
-
-impl std::error::Error for HandlerError {}
-
-impl fmt::Display for HandlerError {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self {
-            HandlerError::Message(msg) => write!(f, "{}", msg),
-            HandlerError::Other(_) => write!(f, "An internal error occurred."),
-        }
-    }
+macro_rules! inform {
+    (msg:literal $(,)?) => {
```

Comment:
This arm is never taken because it expects `msg:literal` literally as you didn't prefix it with `$`. In any case, this whole arm is not really necessary because `expr` includes `literal` basically.